lib/terraform.nix

```diff
@@ -11,6 +11,7 @@ in
 , tfConfig ? null
 , subdir ? defaultSubDir
 , backend ? true
+, vault-addr ? "https://vault.serokell.org:8200"
```

Comment:
Will it be possible to use terranix configuration before `terranix.lib.terranixConfiguration` call to determine the Vault address from there? :thinking: